aggregator/src/metrics.rs

```diff
@@ -341,3 +341,27 @@ pub(crate) fn aggregate_step_failure_counter(meter: &Meter) -> Counter<u64> {
 
     aggregate_step_failure_counter
 }
+
+const TIME_SKEW_HISTOGRAM_VALUES: &[f64] = &[
+    30.0, 60.0, 120.0, 300.0, 600.0, 1800.0, 3600.0, 7200.0, 14400.0, 28800.0, 86400.0,
```

Comment:
Here's a plot of these boundaries on a logarithmic scale:

![image](https://github.com/user-attachments/assets/d66e3693-dc8b-4525-beca-e3e5925502a1)

I like that all the ratios between successive boundaries are 2, 2.5, or 3.